test/test-utils/test-utils.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+FAILED=()
+
+function echoStderr()
+{
+    echo "$@" 1>&2
+}
+readonly -f echoStderr
+
+check() {
```

Comment:
Please apply the pattern of declaring this as a function and marking it readonly to other functions.

```
function myfn() {
...
}
readonly -f myfn
```

Thanks.